src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -365,8 +369,8 @@ const Mentor = ({ dispatch, error, successPage }) => {
                   >
                     {contributionArray.map(area => (
                       <Checkbox
-                        value={area.value}
-                        onValueChange={handleTechStack}
+                        value={area.name}
+                        onChange={e => handleChange(e, 'checkbox')}
```

Comment:
I don't fully understand how this worked before--without the hooks. Thanks for catching this.